packages/system-tests/src/mocha-runner.ts

```diff
@@ -10,11 +10,24 @@ import * as glob from 'glob';
 import * as path from 'path';
 import Mocha = require('mocha');
 
-const mocha = new Mocha({ ui: 'bdd', timeout: 360000, slow: 5000 });
+const mocha = new Mocha({
+  ui: 'bdd',
+  timeout: 360000,
+  slow: 5000,
+  reporter: 'mocha-multi-reporters',
+  reporterOptions: {
+    reporterEnabled: 'mocha-junit-reporter, xunit, spec',
+    mochaJunitReporterReporterOptions: {
+      mochaFile: 'junit-custom.xml'
+    },
+    xunitReporterOptions: {
+      output: 'xunit.xml'
+    }
+  }
+});
```

Comment:
System tests defined its own programmatic way of running the tests so I added the necessary reporter options so it will produce test results as well.